src/test/java/edu/kit/compiler/transform/IRExpressionVisitorTest.java

```diff
@@ -46,7 +42,7 @@ private ExpressionNode getReturnExpressionFromProgram(String input) {
         for (ClassNode classNode: ast.getClasses()) {
             for (MethodNode.DynamicMethodNode m: classNode.getDynamicMethods()) {
                 var mapping = LocalVariableCounter.apply(m);
-                this.context = new TransformContext(
+                new TransformContext(
```

Comment:
Is this call necessary?

Reply:
@0xNB

Reply:
I don't think so. Actually, the whole file seems quite unnecessary to me.